docs/examples/config/example-config.json

```diff
@@ -35,7 +35,10 @@
             "grpc_port": "50051"
         }
     ],
-    "forwarding_cache_timeout": 0.250, // in seconds, could be 0, which means no cache
```

Comment:
This was not released to the public yet, right?

Reply:
No, it was not.